pkg/provisioner/ironic/ironic.go

```diff
@@ -1129,6 +1129,71 @@ func (p *ironicProvisioner) Adopt(force bool) (result provisioner.Result, err er
 	return
 }
 
+func (p *ironicProvisioner) startManualCleaning(ironicNode *nodes.Node) (result provisioner.Result, err error) {
+	var cleanSteps []nodes.CleanStep
+
+	if p.bmcAccess.RAIDInterface() != "" {
+		// Set raid configuration
+		err = setTargetRAIDCfg(p, ironicNode)
+		if err != nil {
+			p.log.Error(err, "failed in setting the target RAID configuration")
+			return
+		}
+		// Build raid clean steps
+		cleanSteps = append(cleanSteps, buildRAIDCleanSteps(p.host.Status.Provisioning.RAID)...)
```

Comment:
Isn't it just
```go
cleanSteps = buildRAIDCleanSteps(p.host.Status.Provisioning.RAID)
```
?

Reply:
No, you need it to pass an (assignable) slice to a variadic function